_layouts/links.html

```diff
@@ -0,0 +1,76 @@
+---
+layout: base
+---
```

Comment:
I was thinking about the design of the page. Maybe something really simple like:

![sketch1683190922813](https://user-images.githubusercontent.com/1842467/236159267-8f029bb8-2d2f-4a78-9767-3a02bf7a79c6.png)

With
- The background color being maybe the green from OLS
- Each box being a button that link directly to the content described
- Short description for each box
- Having an icon (from Font awesome) for each box
- Making the page not too large and making sure it fits best on smartphones (as it will be main use there)

Reply:
> The background color being maybe the green from OLS

Green as a background color looks a bit strange. Screenshot below...

> Each box being a button that link directly to the content described
> Short description for each box
Done. 🙌🏼

> Having an icon (from Font awesome) for each box
Done. Two of the icons are not displaying though. Those are probably for premium users of fontawesome.

Reply:
How do that look like with different background color for the buttons?

Reply:
Sorry I missed this comment earlier. It looks like this.

![Screenshot (491)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/98ba9b0b-4277-4122-8a9f-5d3340143723)

**Also:** I ran this page through wave. The only contrast error was with the `description` section. It looks better when I change the text `color` for OLS description to white

Reply:
How does it look like with description in white? The logo look strange...
Can you also make the width a bit smaller (so the buttons and descriptions are not that large)?

Reply:
How is the width impacting the button text? I do not get that... Maybe sth to modify in the css then

Reply:
Description with white text...
![Screenshot (493)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/0850a0e5-1c9a-48ea-9e3a-18ecb4ddc493)

Description with black text...
![Screenshot (492)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/2f28ca88-1c68-434f-840f-5eb7bd40b2cf)

Reply:
This is what I mean.
The width here is 50%. The font-size will have to be smaller to fit into the button now..

![Screenshot (494)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/c86609f8-a35a-4915-84dd-7269a369f68e)

Reply:
Maybe you can leave the width as % but also defined a max-width so it does not become too large on larger screen

Reply:
Can you check the contrast with white text on green background with https://webaim.org/resources/contrastchecker/? Thanks

Reply:
The two colors (white #FFF and Green #48B87D) failed the test.
I ran the page through Wave as well and got one contrast error.
![Screenshot (495)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/6bec9327-6cde-44d4-a71d-b684ed679d91)